src/Microsoft.ML.DataView/Microsoft.ML.DataView.csproj

```diff
@@ -5,7 +5,12 @@
     <IncludeInPackage>Microsoft.ML.DataView</IncludeInPackage>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <RestoreSources>$(RestoreSources);https://pkgs.dev.azure.com/foundry99/_packaging/SEAL/nuget/v3/index.json</RestoreSources>
```

Comment:
All of ML.NET's dependencies need to be on nuget.org. Our users can't use our packages if the dependency graph isn't available to them.

Reply:
Absolutely, I'm just pending the package getting published on Nuget. I had some issues and delays getting the permissions set correctly, but SEAL should be published tonight or tomorrow morning and I will update the PR.